includes/Modules/Idea_Hub.php

```diff
@@ -172,15 +172,16 @@ public function register_persistent() {
 		add_filter(
 			'display_post_states',
 			function( $post_states, $post ) {
-				if ( 'draft' !== $post->post_status ) {
+				if ( ! $this->is_idea_post( $post->ID ) ) {
 					return $post_states;
 				}
 				$idea = $this->get_post_idea( $post->ID );
-				if ( is_null( $idea ) ) {
-					return $post_states;
+				if ( '' === $post->post_title && 'draft' === $post->post_status ) {
+					/* translators: %s: Idea Hub Idea Title */
+					$post_states['draft'] = sprintf( __( 'Idea Hub Draft “%s”', 'google-site-kit' ), $idea['text'] );
```

Comment:
@felixarntz given that WP already allows for multiple post states and formats them as a list, I feel like we shouldn't overwrite the `draft` state here. Why not use the same `inspired by Idea Hub` here as well? We could still include the idea name/title for this case specifically too. If it was a draft it would look like `Draft, inspired by Idea Hub "{idea-name}"`

Reply:
@aaemnnosttv That makes sense, however I think overwriting the `draft` state here works as well, we essentially combine two into one. While your suggested approach would be a bit cleaner, I think having that one special situation is acceptable as well, which allows us to keep the copy originally defined.